sdk/storage/Azure.Storage.DataMovement/src/TransferManager.cs

```diff
@@ -5,6 +5,7 @@
 using System.Buffers;
 using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.CompilerServices;
```

Comment:
I assume this isn't used?

Reply:
This import is being used for [EnumeratorCancellation]